configure.ac

```diff
@@ -127,6 +127,9 @@ if test -z `which pkg-config 2>/dev/null` ; then
   AC_MSG_ERROR(Cannot find pkg-config)
 fi
 
+bashcompletiondir='${prefix}'`pkg-config --variable=completionsdir bash-completion`
+AC_SUBST(bashcompletiondir)
```

Comment:
What's the portability of this? How does it behave when there is no `bash-completion`? Here, I have:
```
% pkg-config --variable=completionsdir bash-completion
Package bash-completion was not found in the pkg-config search path.
Perhaps you should add the directory containing `bash-completion.pc'
to the PKG_CONFIG_PATH environment variable
No package 'bash-completion' found
zsh: exit 1     pkg-config --variable=completionsdir bash-completion
```

Reply:
The new version should behave correctly when the package is not found (a default path is given).